tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/IntegratedWindowsAuthUsernamePasswordTests.cs

```diff
@@ -278,6 +282,10 @@ public void AcquireTokenByIntegratedWindowsAuthTest_UnknownUser()
         [DeploymentItem(@"Resources\WsTrustResponse13.xml")]
         public async Task AcquireTokenByIntegratedWindowsAuthTestAsync()
         {
+            IDictionary<string, string> extraQueryParamsAndClaims =
+                MsalTestConstants.ExtraQueryParams.ToDictionary(e => e.Key, e => e.Value);
```

Comment:
don't need ToDictionary() here (extraqueryparams is already a dictionary)

Reply:
it's an IDictionary. In any case, the code here is meant to clone the dictionary.